### ISSUE TITLE
Archive cronjob is created despite setting 'enabled: false' in Values.yaml

### ISSUE THREAD

**Issue by ashwin027** [bitnami/charts#20567]
### Name and Version

bitnami/matomo 3.0.2

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. install matomo
2. Run the command to check the cronjobs that were created

`kubectl get cronjob`

3. The archive cronjob is created when the default values in the values.yaml file has cronjob-->archive-->enabled set to false,

cronjob:
  ...
  archive:
    ## @param cronjobs.archive.enabled Whether to enable scheduled mail-to-task CronJob
    ##
    enabled: false
    ## @param cronjobs.archive.schedule Kubernetes CronJob schedule
    ##
    schedule: "*/5 * * * *"
    ## @param cronjobs.archive.suspend Whether to create suspended CronJob

### Are you using any custom parameters or values?

No, I'm using the default values.

### What is the expected behavior?

The Archive Cron job should not be created.

### What do you see instead?

The Cronjob is created.

**Comment by fmulero** [comment]
Good catch!, Thanks a lot @ashwin027 for reporting this issue. I think each job should have its own flag to enable or disable it. Would you like to contribute by creating a PR to solve the issue? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md) you can find the contributing guidelines.
